release.sh

```diff
@@ -15,32 +15,27 @@ if [[ ! $(command -v jq) ]]; then
 	exit 1
 fi
 
-if [[ ! -z $(git status -s) ]]; then
-    echo "Working tree not clean. Please re-run when the tree is clean."
-    exit
-fi
-
-local_rev=`git rev-parse HEAD`
-remote_rev=`curl -s https://api.github.com/repos/influxdata/vsflux/commits/master | jq -r .sha`
-if [[ $local_rev != $remote_rev ]]; then
-    echo "Local HEAD $local_rev is not the same as remote rev $remote_rev"
-    exit
-fi
+TEMPDIR=$(mktemp -d -t flux-release.XXXX)
+echo "Using fresh install in $TEMPDIR"
+cd $TEMPDIR
+git clone git@github.com:influxdata/vsflux.git > /dev/null 2>&1
+cd $TEMPDIR/vsflux
 
 npm add @influxdata/flux-lsp-node
 if [[ ! -z $(git status -s) ]]; then
     echo "Please upgrade flux-lsp-node to the latest version prior to release."
+    rm -rf $TEMPDIR
     exit
 fi
 
 new_version=`npm version patch --sign-git-tag`
```

Comment:
What does this do? It must do something since the next line does a `push`.

Reply:
It updates the `package.json` file, commits that, and then makes a signed tag from that commit. It's one of those things where your git tag is the version, but also, your `package.json` is the _actual_ version, and you want those two things to align.